order/src/main/java/org/order/OrderService.java

```diff
@@ -0,0 +1,34 @@
+package org.order;
+
+import jakarta.transaction.Transactional;
+import java.util.List;
+import java.util.stream.Collectors;
+import lombok.AllArgsConstructor;
+import org.order.dto.PurchaseOrderDto;
+import org.order.entity.PurchaseOrder;
+import org.springframework.stereotype.Service;
+
+@Service
+@AllArgsConstructor
+public class OrderService {
+
+    private final OrderMapper orderMapper;
+    private final PurchaseOrderRepository purchaseOrderRepository;
+
+    @Transactional
+    Long addOrder(PurchaseOrderDto purchaseOrderDto) {
+        PurchaseOrder purchaseOrder = purchaseOrderRepository.save(orderMapper.mapOrderDtoToEntity(purchaseOrderDto));
+        return purchaseOrder.getPurchaseOrderId();
+    }
+
+    @Transactional
```

Comment:
You can be on safe side and use readOnly = true
```suggestion
    @Transactional(readOnly = true)
```